docs/product/components/popovers.html

```diff
@@ -26,21 +26,80 @@
+                    <td class="p8">A unique id that the popover’s originating element can target.</td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</section>
 <section class="stacks-section">
     {% header h2 | Examples %}
-    <div class="mb24 p16 bar-sm bg-black-025 ba bc-black-1">
-        <div class="ps-relative">
-            <div class="s-popover ps-absolute ws2 b48 l64 mln4 js-popover-example" role="menu" aria-hidden="true">
-                <div class="s-popover--arrow__bl"></div>
-                <p class="ff-mono mb8">.s-popover--arrow__bl</p>
-                <p class="mb0">Popover arrow appears bottom left</p>
+    <div class="mb24 p16 bar-sm bg-black-025 ba bc-black-1 ps-relative">
+        <button class="s-btn s-btn__filled"
+            data-controller="s-popover"
+            data-action="s-popover#toggle"
+            aria-haspopup="true"
+            aria-expanded="false"
+            aria-controls="example-popover">
+            Show example popover
+        </button>
+
+        <div class="s-popover ps-absolute" id="example-popover" role="menu" aria-hidden="true">
+            <div class="s-popover--arrow__bc"></div>
+            <div tabindex="0">
+                <p class="ff-mono mb8">.s-popover--arrow__bc</p>
+                <p class="mb0">Popover arrow appears bottom center</p>
             </div>
-            <button class="s-btn s-btn__filled js-popover-btn">Show example popover</button>
         </div>
-    </div>
-    <p class="stacks-copy">Popovers are provided without any positioning applied to them. It’s up to the implementer to choose how they’ll be positioned. This will most likely be achieved by adding something like <code class="stacks-code">ps-absolute t8 l8</code> to <code class="stacks-code">.s-popover</code>. Positioning for arrows <em>is</em> provided, however. Arrow direction (top, right, bottom, left) will appear first, followed by the secondary (center, top, right, bottom, left). For example, a popover with an arrow child of <code class="stacks-code">.s-popover--arrow_tc</code> will appear on the top of the <code class="stacks-code">.s-popover</code> and centered horizontally. Though there are sensible defaults applied to the width of popovers, you may need to adjust manually.</p>
 
-    <p class="stacks-copy">By default, popovers are hidden. Adding the class <code class="stacks-code">.is-visible</code> will show the popover.</p>
+        <button class="s-btn s-btn__filled"
+            data-controller="s-popover"
+            data-action="s-popover#toggle"
+            aria-haspopup="true"
+            aria-expanded="false"
+            aria-controls="second-example-popover">
+            Show second example popover
+        </button>
+
+        <div class="s-popover ps-absolute" id="second-example-popover" role="menu" aria-hidden="true">
+            <div class="s-popover--arrow__lb"></div>
+            <div tabindex="0">
```

Comment:
This adds a nasty focus ring when interacting via the mouse. Is there a way we can only add the focus ring when the user tabs to the popover? And perhaps we put this on the popover itself. Happy to provide custom `:focus` classes for this case.

Reply:
We could just remove the `tabindex`  It was just meant as an alternative to putting dummy button or something in there.  I did choose a new div because I knew adding it to the root element would offend you more:

![Screen Shot 2019-04-02 at 3 13 10 PM](https://user-images.githubusercontent.com/662837/55440241-e7272d00-555b-11e9-8bbb-31d751ff88d4.png)